README.md

```diff
@@ -1,2 +1,82 @@
 # queries
 Hosts reusable log queries which are built into a single queries.json file.
+
+
+# Query database structure
+
+```yaml
+queries:
+  query1:  # that is query id (sorted by id)
+    name: foo  # required
```

Comment:
why is name required?
I'd like to have more meaningful example here without all these "query1 foo bar aaa". It doesn't explain.

Reply:
We want to force people adding queries to give them meaningful names, the name is effectively what we will display when the patterns are identified, what will be clearly visible in the sova html report from each build.

Still, if you think that this adds undesired complexity, we could re-use the "id" (key name) as a name when is missing. Lets see what others think.

Reply:
OK, so this is "message" in sova, I think "message" gives a more clear understanding.